common/app/controllers/EmailSignupController.scala

```diff
@@ -488,6 +559,70 @@ class EmailSignupController(
     }
   }
 
+  // TO DO - deduplicate with submit method
+  def submitMany(): Action[AnyContent] =
+    Action.async { implicit request =>
+      AllEmailSubmission.increment()
+
+      emailFormManyNewsletters
+        .bindFromRequest()
+        .fold(
+          formWithErrors => {
+            log.info(s"Form has been submitted with errors: ${formWithErrors.errors}")
+            EmailFormError.increment()
+            Future.successful(respond(InvalidEmail))
+          },
+          form => {
+            log.info(
+              s"Post request received to /email/many/ - " +
+                s"email: ${form.email}, " +
```

Comment:
Aware that we already have a lot of sensitive data in our logs, but not keen on increasing it if we don't have to! Do we need to include emails in this log entry?

Reply:
I don't see any need for this on my end - I copied the log pattern from the old method. It would be good to remove the email addresses from the existing log statements in the controller too.

Reply:
I've pushed the changes to remove it - I also took out the recaptcha token - I don't think there's a good reason to log this.